### PR TITLE
modify UI design

### PR DESCRIPTION
**Pull request by kimgayeon**
align get ticket button to center
change app and calendar color
add location on place image
add class name and expert name on class image

### CODE REVIEW

File: Android/app/src/main/java/com/example/kahye/common/Adapter.java [hunk 1]
```diff
@@ -2,6 +2,7 @@
```
Comment (seunglee1b) [inline Android/app/src/main/java/com/example/kahye/common/Adapter.java]: @Kahyelee I see the name of this file contains your name. Is this the thing that you brought up yesterday when we had 1:1?

File: Android/app/src/main/java/com/example/kahye/common/Adapter.java [hunk 2]
```diff
@@ -23,29 +24,33 @@
...
     private String [] classes = {};
+    private String[] expertNameList = {};
 
-    public Adapter(Context context, ClassList classList, String[] imagesURL, String selectedDate) {
+    public Adapter(Context context, ClassList classList, String[] imagesURL,
+                   String[] expertNameList, String selectedDate) {
         this.context = context;
         this.classList = classList;
         this.imagesURL = imagesURL;
+        this.expertNameList = expertNameList;
```
Comment (seunglee1b) [inline Android/app/src/main/java/com/example/kahye/common/Adapter.java]: @Kahyelee I just started to take a look at this. Why do we have **classList**, **imagesURL** and **expertNameList** here? I thought that class object contains its corresponding image and an instructor name.

File: Android/app/src/main/java/com/example/kahye/common/network/RetrofitInstance.java [hunk 3]
```diff
@@ -9,7 +9,8 @@
     private static Retrofit retrofit;
     private static OkHttpClient client;
     //URL can be easily changed
-    private static final String BASE_URL = "http://10.0.2.2:8000/api/v1/";
+    private static final String BASE_URL =
+            "http://52.8.187.167:8000/api/v1/";
```
Comment (seunglee1b) [inline Android/app/src/main/java/com/example/kahye/common/network/RetrofitInstance.java]: I think we can make another module that takes care of a network call. So, this file can import that module and pass "/api/v1" as a parameter and have that module make a network call.

File: Android/app/src/main/java/com/example/kahye/common/Adapter.java (hunk 1 above)
Comment (Kahyelee) [reply Android/app/src/main/java/com/example/kahye/common/Adapter.java]: Yes It is. I tried to change it but we had to do other things first. I'll fix it sooner.

File: Android/app/src/main/java/com/example/kahye/common/Adapter.java (hunk 2 above)
Comment (Ricci0708) [reply Android/app/src/main/java/com/example/kahye/common/Adapter.java]: I changed the code. The classList variable is the list of classes, so it is needed to show the information about the class in adapter. I deleted the imagesURL and expertNameList variables.

**Comment by kimgayeon** [comment]
@ethkim  @seunglee1b 
I modify UI design.

class info design
text style bold
change date on calendar
merge with reservation type 2 code

File: Android/app/src/main/java/com/example/kahye/common/Adapter.java (hunk 2 above)
Comment (Ricci0708) [reply Android/app/src/main/java/com/example/kahye/common/Adapter.java]: I changed the code in my local if the request is merged I will make pull request about this issue #24

File: Android/app/src/main/java/com/example/kahye/common/ReservationActivity.java [hunk 4]
```diff
@@ -49,39 +58,53 @@ protected void onCreate(Bundle savedInstanceState) {
...
         ArrayAdapter<String> adapter = new ArrayAdapter<>(this,
                 android.R.layout.simple_list_item_1, timeList);
 
         List<TimeTable>  timeslot = selectedClass.getAvailableTimeTable();
         for (int timeListIdx = 0; timeListIdx < timeslot.size(); timeListIdx++){
-            String timeString = timeslot.get(timeListIdx).getStartTime().toString() + " ~ " +
-                    timeslot.get(timeListIdx).getEndTime().toString();
+            String timeString = timeslot.get(timeListIdx).getStartTime().
+                    toString() + " ~ " + timeslot.get(timeListIdx).getEndTime
+                    ().toString();
```
Comment (Ricci0708) [inline Android/app/src/main/java/com/example/kahye/common/ReservationActivity.java]:
```
            String timeString = 
                    timeslot.get(timeListIdx).getStartTime().toString() + " ~ " 
                    + timeslot.get(timeListIdx).getEndTime().toString();
```
How about change the code like this?
Two codes are same but the above one is easy to read

File: Android/app/src/main/java/com/example/kahye/common/TrendingClassActivity2.java [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+    Adapter adapter;
+    ViewPager classViewPager;
+    DatePicker datePicker;
+    ImageButton classButton;
+    String selectedDate;
+
+    String[] imagesURL = {};
+    String[] classes = {};
+    String[] expertNameList = {};
+
```
Comment (Ricci0708) [inline Android/app/src/main/java/com/example/kahye/common/TrendingClassActivity2.java]: Same issue #24  here, I will edit this code too.

File: Android/app/src/main/java/com/example/kahye/common/network/RetrofitInstance.java (hunk 3 above)
Comment (Ricci0708) [reply Android/app/src/main/java/com/example/kahye/common/network/RetrofitInstance.java]: We don't understand exactly what you mean. What about talk about this in person?

File: Android/app/src/main/java/com/example/kahye/common/ReservationActivity.java (hunk 4 above)
Comment (kimgayeon) [reply Android/app/src/main/java/com/example/kahye/common/ReservationActivity.java]: ok. I will change that code :)

**Pull request merged**